UnitsNet/CustomCode/UnitAbbreviationsCache.cs

```diff
@@ -215,7 +215,7 @@ public string GetDefaultAbbreviation(Type unitType, int unitValue, IFormatProvid
         /// <param name="unit">Enum value for unit.</param>
         /// <param name="formatProvider">The format provider to use for lookup. Defaults to <see cref="CultureInfo.CurrentCulture" /> if null.</param>
         /// <returns>Unit abbreviations associated with unit.</returns>
-        public string[] GetUnitAbbreviations<TUnitType>(TUnitType unit, IFormatProvider? formatProvider = null) where TUnitType : Enum
+        public string[] GetUnitAbbreviations<TUnitType>(TUnitType unit, IFormatProvider? formatProvider = null) where TUnitType : struct, Enum
         {
             return GetUnitAbbreviations(typeof(TUnitType), Convert.ToInt32(unit), formatProvider);
         }
```

Comment:
This is great, introducing the `struct` constraint here opens up the possibility for using 
`Unsafe.As<TUnit, int>(ref unit)`
instead of 
`Convert.ToInt32(unit)` 
which is ~34x faster on `net8.0` and ~92x faster on `net48` 🚀

Reply:
Sounds good!